packages/react/src/field/FieldHint.tsx

```diff
@@ -1,12 +1,12 @@
+import { PropsWithChildren } from 'react';
 import { Text } from '../text';
 
-export const FieldHint = ({
-	children,
-	id,
-}: {
-	children: string;
-	id?: string;
-}) => (
+export type FieldHintProps = PropsWithChildren<{
```

Comment:
For consistency, I've added prop comments here and exported the type while I was touching these files.